modules/nf-core/yahs/meta.yml

```diff
@@ -51,19 +51,19 @@ output:
   - scaffolds_fasta:
       type: file
       description: FASTA file with resulting contigs
-      pattern: "*scaffolds_final.fa"
+      pattern: "test_scaffolds_final.fa"
```

Comment:
No, you were right before- these should hold the wildcards. It's test.yml needs fixing.